.gitmodules

```diff
@@ -4,9 +4,9 @@
 	branch = ufs_public_release
 [submodule "ccpp/framework"]
 	path = ccpp/framework
-	url = https://github.com/NCAR/ccpp-framework
-	branch = ufs_public_release
+	url = https://github.com/cliimbfuji/ccpp-framework
```

Comment:
I can not checkout this pull request. The URL of this submodule is incorrect. Misspelled '**cliimbfuji**' ?